apps/azureserviceoperator-system/preview/base/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../sealed-secrets
```

Comment:
Do we need this folder `sealed-secrets` or can we just add into preview/base?

Reply:
Yeh sure, I'll tidy this. I was using OSBA as a sort of reference.